external-crates/move/crates/move-compiler/src/unit_test/plan_builder.rs

```diff
@@ -555,12 +550,12 @@ fn parse_gas_budget_attribute(context: &mut Context, gas_budget_attr: Option<&E:
                     TestingAttribute::GAS_BUDGET_HEAP_SIZE => {
                         let u = get_assigned_attribute_as_u64(context, TestingAttribute::GAS_BUDGET_HEAP_SIZE, attr_loc, attr)?;
                         // TODO: Do some sanity check that u shouldn't be larger than a max value.
-                        assigned_gas_budget.heap_size = u;
+                        assigned_gas_budget.heap_size = u as usize;
```

Comment:
Here in each case we repeat a long constant name, I think Rust allows this syntax instead:
```Rust
                    x if x == TestingAttribute::GAS_BUDGET_HEAP_SIZE => {
                        let u = get_assigned_attribute_as_u64(context, x, attr_loc, attr)?;
```